src/test/java/com/wikia/webdriver/testcases/renametool/RenameToolTests.java

```diff
@@ -100,19 +101,20 @@ public void ConfirmationModalDecline_RedirectionToRenameTool() {
   }
 
   @Test
-  public void NewUserCreateAndRenameDone() {
+  public void newUserCreateAndRenameDone() {
     Credentials credentials = new Credentials();
     String timestamp = Long.toString(DateTime.now().getMillis());
+    String hashstamp = UUID.randomUUID().toString().replace("-","0");;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this empty statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)